src/app/component/Slider/Slider.component.js

```diff
@@ -35,12 +33,12 @@ import './Slider.style';
 export class Slider extends PureComponent {
     static propTypes = {
         showCrumbs: PropTypes.bool,
+        mix: MixType,
         activeImage: PropTypes.number,
         onActiveImageChange: PropTypes.func,
-        mix: MixType,
         children: ChildrenType.isRequired,
+        childrenClones: ChildrenType.isRequired,
```

Comment:
Is it possible to clone them inside of the slider?
From the names I think that this is a derivative from `children`, why not?

Reply:
Because of the different keys.

Reply:
...?